apps/src/lib/levelbuilder/rubrics/LearningGoalItem.jsx

```diff
@@ -0,0 +1,108 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import color from '@cdo/apps/util/color';
+import {borderRadius} from '@cdo/apps/lib/levelbuilder/constants';
+import EvidenceDescriptions from './EvidenceDescriptions';
+import Button from '../../../templates/Button';
+
+export default function LearningGoalItem({deleteItem, id}) {
+  const [aiEnabled, setAiEnabled] = useState(false);
+
+  const handleCheckboxChange = () => {
+    setAiEnabled(!aiEnabled);
+  };
+
+  return (
+    <div className="uitest-learning-goal-card">
+      <div
+        style={{
+          ...styles.activityHeader,
+        }}
+      >
+        <div style={styles.activityHeaderComponents}>
+          <div style={styles.inputsAndIcon}>
+            <div>
+              <label style={styles.labelAndInput}>
+                <span style={styles.label}>{`Key Concept:`}</span>
+                <input
+                  style={{width: 600}}
+                  className="uitest-rubric-key-concept-input"
+                />
+              </label>
+            </div>
+            <label>
+              Use AI to assess
+              <input
+                type="checkbox"
+                checked={aiEnabled}
+                onChange={handleCheckboxChange}
+                style={styles.checkbox}
+              />
+            </label>
+          </div>
+        </div>
+      </div>
+      <div style={styles.activityBody}>
+        <EvidenceDescriptions isAiEnabled={aiEnabled} />
+        <Button
+          text="Delete key concept"
+          color={Button.ButtonColor.red}
+          onClick={() => deleteItem()}
+          icon="trash"
+          iconClassName="fa-trash"
+        />
+      </div>
+    </div>
+  );
+}
+
+LearningGoalItem.propTypes = {
+  checked: PropTypes.bool,
```

Comment:
I don't see this in the props list?

Reply:
No - this was left over when I had a different setup for the "AI enabled?" checkbox.  I will remove it.